feedbacks.js

```diff
@@ -30,88 +30,83 @@ export function getFeedbacks() {
 			},
 		],
 		callback: (feedback) => {
-			if (this.states.playing === feedback.options.playPause) {
-				return true
-			}
+			return this.states.playing === feedback.options.playPause
 		},
 	}
 
 	feedbacks['playingCueName'] = {
 		type: 'boolean',
-		name: 'Change style if specific cue name is playing',
+		name: 'Cue name is playing',
 		description: 'Change style based on play status of cue name',
 		defaultStyle: {
 			color: ColorWhite,
 			bgcolor: ColorGreen,
 		},
 		options: [
 			{
-				type: 'textwithvariables',
+				type: 'textinput',
+				useVariables: true,
 				label: 'Cue Name',
 				id: 'cueName',
 				default: '',
 			},
 		],
-		callback: (feedback) => {
-			this.parseVariablesInString(feedback.options.cueName, (value) => {
-				feedback.options.cueName = value
-			})
-			if (this.states.playing == 'Playing' && this.states.currentCueName == feedback.options.cueName) {
-				return true
-			}
+		callback: async (feedback) => {
+			const value = await this.parseVariablesInString(feedback.options.cueName)
```

Comment:
```
callback: async (feedback, context) => {
			const value = await context.parseVariablesInString(feedback.options.cueName)
```
You should do this, so that the feedback can be re-checked when the variables being parsed change

Reply:
thanks for the advice. fixed with https://github.com/bitfocus/companion-module-imimot-mitti/pull/38